.travis.yml

```diff
@@ -75,6 +78,7 @@ before_script:
   ## icub-main
   - git clone https://github.com/robotology/icub-main.git
   - cd icub-main
+  - git checkout v1.13.0
```

Comment:
Why the version bump? or downgrade?

Reply:
https://github.com/robotology/icub-model-generator/pull/119#issuecomment-531726813